Lab3.ipynb

```diff
@@ -0,0 +1,405 @@
+      "text/plain": [
+       "<PIL.JpegImagePlugin.JpegImageFile image mode=RGB size=283x45 at 0x7F40B776E790>"
+      ]
+     },
+     "execution_count": 118,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# 2.2: \n",
+    "print('5 sigma probability:', str(1-stats.norm.cdf(5)))\n",
+    "Image.open('Lab3.2.jpg')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Problem 2.2: Above is the probability of 5 sigma for a normal gaussian distribution, which translates to the integral of the pdf from a value x out to infinity. Since we want to know the distance corresponding to 5 sigma, we should look for the bounding value x. Since we are looking radially in 2-d space this distance isn't too difficult to calculate. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 119,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "d = 5.489\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"d =\", 5.489)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Problem 3.1: What is the probability that the background data produces 6800 or less comsmic rays in 15 days?"
```

Comment:
Problem 3.1:
- typo: comsmic --> cosmic
- I am unsure if this is the correct statistical question. I guess I don't really know where it leads based on what is currently written, and wouldn't know how to translate it into the math based on that.
- (We got the same numeric answers, though, so we followed the same math.) I had to read the problem several times to understand what it's asking, but I think (? I hope) that by calculating the "significance of the moon shadow detection", we are actually looking at the significances of higher energy cosmic rays in our data (this is talked about in the first descriptive paragraph of problem 3, which causes the "moon shadow detection" due to absorption). I also think that this is what the problem means by "unique aspects" since we're looking at cosmic rays which appear to be missing from the background due to the explanation in the lab instructions. I think this is the "unique" aspect because we normally look at the presence of data and not absence of it. 

(I hope this is the right line of thinking)